src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
```

Comment:
Prefer `const` over `let` for the `result` variable if it's not being reassigned, to prevent unintentional reassignments as per checklist item 3.